Kira_sqrt_stub.R

```diff
@@ -0,0 +1,31 @@
+mysqrt <- function( x=81, tol=.00001, myguess=10, verbose=FALSE){		#begining the function with starting parameters
+	if(x<0) {						#creates absolute value function so it will work with negative numbers
+		x <- abs(x)
+	}
```

Comment:
No! You canʻt just pretend itʻs positive. This will return an incorrect answer. The square root of a negative number is an imaginary number.  -1